.github/workflows/zcashd-manual-deploy.yml

```diff
@@ -37,6 +37,17 @@ jobs:
         with:
           short-length: 7
 
+      # Performs formatting on the Zcash network name.
+      #
+      # Labels in GCP are required to be in lowercase, but the blockchain network
+      # uses sentence case, so we need to downcase ${{ inputs.network }}.
+      #
+      # Passes ${{ inputs.network }} to subsequent steps using $NETWORK env variable.
+      - name: Format network name for labels
```

Comment:
Nitpick:

```suggestion
      # Makes the Zcash network name lowercase.
      #
      # Labels in GCP are required to be in lowercase, but the blockchain network
      # uses sentence case, so we need to downcase ${{ inputs.network }}.
      #
      # Passes ${{ inputs.network }} to subsequent steps using $NETWORK env variable.
      - name: Downcase network name for labels
```

Reply:
Thanks, I think I accidentally overwrote that change with a force-push

Reply:
> Thanks, I think I accidentally overwrote that change with a force-push

I think it happened again; I'll re-commit the suggestion.